js/tests/unit/scrollspy.js

```diff
@@ -3,6 +3,22 @@ $(function () {
 
   QUnit.module('scrollspy plugin')
 
+  // To keep unit tests consistent with DRY principles
+  function testElementIsActiveAfterScroll(element, target, assert) {
+    var deferred = $.Deferred()
+    var $target = $(target)
+    var $content = $target.parent()
+    var scrollHeight = Math.ceil($content.scrollTop() + $target.position().top)
+    var done = assert.async()
+    $content.one('scroll', function () {
+      assert.ok($(element).hasClass('active'), 'target:' + target + ', element: ' + element)
+      done()
```

Comment:
I don't understand why `done` is called here. My understanding is that this function is run multiple times within one unit test, `done` should only be called when the unit test is finished.